### PR TITLE
Create Facade Device for Valve and FastValve Device Classes

### LINKED ISSUE #1: Using the provided materials create Valve device class based on the facadedevice library.
Properties:
- [x] PlcAttrName_CloseC
- [x] PlcAttrName_ClosedS
- [x] PlcAttrName_OpenC
- [x] PlcAttrName_OpenS
- [x] PlcAttrName_InterlockA
- [x] PlcAttrName_CutOffA
- [x] PlcAttrName_UnexpectedState

Attributes:
- [x] ValveOpen: PlcAttrName_OpenS readout.
- [x] ValveClosed: PlcAttrName_CloseC readout.
- [x] ValveInterlock: PlcAttrName_InterlockA readout.
- [x] ValveCutOff: PlcAttrName_CutOffA readout.
- [x] UnexpectedState: PlcAttrName_UnexpectedState readout.

Commands:
- [x] Open: write True to PlcAttrName_OpenC.
- [x] Close: write True to PlcAttrName_CloseC.

### LINKED ISSUE #2: Using the provided materials create FastValve device class based on the facadedevice library.
Properties:
- [x] PlcAttrName_CloseC
- [x] PlcAttrName_ClosedS
- [x] PlcAttrName_OpenC
- [x] PlcAttrName_OpenS
- [x] PlcAttrName_InterlockA
- [x] PlcAttrName_CutOffA
- [x] PlcAttrName_UnexpectedState
- [x] PlcAttrName_Inrush1A
- [x] PlcAttrName_Inrush2A

Attributes:
- [x] ValveOpen: PlcAttrName_OpenS readout.
- [x] ValveClosed: PlcAttrName_CloseC readout.
- [x] ValveInterlock: PlcAttrName_InterlockA readout.
- [x] ValveCutOff: PlcAttrName_CutOffA readout.
- [x] UnexpectedState: PlcAttrName_UnexpectedState readout.
- [x] InRush1A:  PlcAttrName_Inrush1A readout.
- [x] InRush2A:  PlcAttrName_Inrush2A readout.

Commands:
- [x] Open: write True to PlcAttrName_OpenC.
- [x] Close: write True to PlcAttrName_CloseC.

### PR DESCRIPTION
**Pull request by stanislaw47**
Closes #1, closes #2 
* added project structure, able to run to different devices on one device server
* added two device classes: `Valve` and `FastValve`
* added automatically build documentation
* added `LICENSE`, updated `README`

### CODE REVIEW

File: valve_ds/fast_valve.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+class FastValve(Valve):
+    """
+
+    InrushA1 should be True if air-inrush is detected on first sensor or False
+    in normal state.
+    InrushA2 should be True if air-inrush is detected on second sensor or False
+    in normal state.
+    """
+    __doc__ = Valve.__doc__ + __doc__
```
Comment (LJBD) [inline valve_ds/fast_valve.py:19]: Have you tried building the documentation with such commands? Does it work?

File: valve_ds/fast_valve.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        :param cutoff: ValveCutOff
+        :param opn: ValveOpen
+        :param closed: ValveClosed
+        :param unexp: UnexpectedState
+        :param rush1: InRush1A
+        :param rush2: InRush2A
+
+        :return: appropriate device state and status
+
+        :rtype: DevState
```
Comment (LJBD) [inline valve_ds/fast_valve.py]: You actually return a tuple: (DevState, string)

File: valve_ds/valve.py [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+        :param interlock: ValveInterlock
+        :param cutoff: ValveCutOff
+        :param opn: ValveOpen
+        :param closed: ValveClosed
+        :param unexp: UnexpectedState
+
+        :return: appropriate device state and status
+
+        :rtype: DevState
```
Comment (LJBD) [inline valve_ds/valve.py]: You actually return a tuple: (DevState, string)

**Review by LJBD — CHANGES_REQUESTED** [review]

File: valve_ds/fast_valve.py (hunk 1 above)
Comment (stanislaw47) [reply valve_ds/fast_valve.py:19]: Yes, it does, however only with class docstring. I've tried that with `state_and_status_fast_valve` method docstring and it hasn't worked out.

**Review by LJBD — APPROVED** [review]

**Pull request merged**